src/chessground/fen.ts

```diff
@@ -27,6 +27,36 @@ const letters = {
   king: 'k'
 }
 
+export function readPeripheral(fen: string): cg.PeripheralPieces {
```

Comment:
Maybe parsePeripherialFen

Reply:
Hmm, here I'm trying to be consistent with function:
``` 
export function read(fen: string): cg.Pieces {
```

Reply:
I would rename read as well

Reply:
Fixed